cache/cache.go

```diff
@@ -0,0 +1,15 @@
+package cache
```

Comment:
Maybe we should have smaller interfaces and compose into one Cache interface and one TTLCache interface.

Reply:
Done; I broke up the interface into two new ones. 
- `lru` satisfies the *Cache* interface, and
- `redis` satisfies both the *Cache* and *TTLCache* interfaces.

How does it look?